### ISSUE TITLE
Allow CeleryPingHealthCheck to check user-specific queues

### ISSUE THREAD

**Issue by BoPeng** [KristianOellegaard/django-health-check#428]
`CeleryPingHealthCheck` checks if some pre-defined queues have active workers

https://github.com/revsys/django-health-check/blob/251b1562c1a7b4bb95b46066fbde0bfaac41567b/health_check/contrib/celery_ping/backends.py#L53

Upon checking, we found that the `CELERY_QUEUES` setting is defined but is empty for our production server. Because this setting is now called [`task_queues`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-task_queues) and its documentation suggests against specifying this setting and use automatic routing, we do not want to mess with this option for our server. Can `django-health-check` allow a setting for a list of queues to check?

**Comment by sevdog** [comment]
This is related to #374.

**Comment by BoPeng** [comment]
Yes, but I was expecting a separate setting that allows us to bypass celery configuration since `app.conf.CELERY_QUEUES` is empty if auto-routing is used. I mean, the production server should be flexible enough to dispatch to available workers automatically but healthcheck should alert us when some workers and/or queues are unavailable.

**Comment by codingjoe** [comment]
Hi there 👋,

Thank you for reaching out. I am currently very hesitant to add more settings. Especially for contrib features. The default implementation should be pretty sufficient for most scenarios.

If you need a customization, I would encourage you to use inheritance to apply the changes you need in your project.

Don't hesitate to reopen this issue, but I will close it for now.

Thanks,
Joe
